proposals/model-and-field-resources.md

```diff
@@ -0,0 +1,256 @@
+
+<br />
+
+# Table of Contents
+- [Summary of Changes](#summary-of-changes)
+- [Section 1: Introduction](#section-1-introduction)
+- [Section 2: Specification](#section-2-specification)
+- [Section 3: Certification](#section-3-certification)
+- [Section 4: Contributors](#section-4-contributors)
+- [Section 5: References](#section-5-references)
+- [Section 6: Appendices](#section-6-appendices)
+- [Section 7: License](#section-7-license)
+
+<br />
+
+# Summary of Changes
+* Adds support for advertising capabilities such as searchability and updatability in a standard format.
+* Adds a new resource called Model, which can be a resource or shape, and adds other information to top-level models like Definition.
+* ResourceName has become ModelKey. Providers can still use ResourceName, but ModelKey must be present.
+* DisplayName has been deprecated in favor of the internationalized DisplayNames collection.
+
+<br />
+
+# Section 1: Introduction
+In order to make server metadata richer, easier to work with, and more Data Dictionary friendly and transport agnostic, this proposal adds support for a new Model Resource and additions to the Field Resource. 
+
+The term "Resource" is used the same way the Data Dictionary uses it now, but there can be other kinds of models as well so the terminology has been generalized.
+
+This proposal works with [Web API Core](https://github.com/RESOStandards/transport/blob/rcp-39-web-api-core-2-1-0/web-api-core.md) standards as well as [RESO Common Format](./reso-common-format.md), and in the latter case a non-OData server could advertise their metadata in standard format using the Model, Field, and Lookup Resources.
+
+<br />
+
+# Section 2: Specification
+This specification adds the Model Resource and makes some additions and changes to the existing Field Resource, as currently defined in Data Dictionary 1.7 and 2.0. 
+
+## 2.1: Field Resource
+
+The following fields will be **removed** from the Field Resource:
+* **[ResourceName](https://ddwiki.reso.org/pages/viewpage.action?pageId=395529)** will be deprecated in favor of **ModelKey**. 
+* **[DisplayName](https://ddwiki.reso.org/pages/viewpage.action?pageId=395531)** will be deprecated in favor of internationalization and locale support being added in #67 with the DisplayNames field.
+
+Providers MAY still use these fields for backwards compatibility, if needed, but the standard versions of these items MUST also be present in that case. Deprecated items will be treated as local in RESO Analytics.
+
+The following fields will be **added** to the [Field Resource](https://ddwiki.reso.org/display/DDW20/Field+Resource):
+
+* **LookupName**
```

Comment:
@psftc - should these field changes/additions be in a table? We usually put them in one in other specs but it seems like listing them is also readable?